<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidgetsClass</class>
 <widget class="QMainWindow" name="MainWidgetsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidgets</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>381</width>
      <height>281</height>
     </rect>
    </property>
    <widget class="QPushButton" name="showBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>显示数据</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>增加数据</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>删除数据</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>修改数据</string>
     </property>
    </widget>
    <widget class="QPushButton" name="findBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>查找数据</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
